multi-int-store/src/test/java/com/connexta/multiintstore/SolrTests.java

```diff
@@ -326,13 +335,63 @@ public String getFilename() {
   }
 
   @Test
-  @Ignore("TODO verify id matches something in S3 before storing to solr")
-  public void testStoreMetadataProductIdNotFound() {
-    // TODO verify 404? 400?
+  public void testStoreMetadataProductIdNotFound() throws Exception {
+    when(mockAmazonS3.doesObjectExist(anyString(), anyString())).thenReturn(Boolean.TRUE);
```

Comment:
Why do you stub this method twice?

Reply:
That was a mistake. The stub at the beginning of the method isn't necessary. I'll put up a PR to address your comments.